### PR TITLE
update: roadmap May 21

### PR DESCRIPTION
**Pull request by p7nov**
Note that there are no links to new items yet.

### CODE REVIEW

**Review by katia-energizer — CHANGES_REQUESTED** [review]
Small formatting changes

**Comment by p7nov** [comment]
> Can add comments to unchanged sections.
> 
> 1. Roadmap will be updated every 6 months (not three months). With potential updates in-between major updates.
> 2. There should be a separate changelog. The first changelog is What's changed since October 2020, the second changelog is What's changed since January 2021.
> 3. Compiler core is missing postponed items in the roadmap. Is it expected?

1. Fixed
2. Not sure we need all changelogs between roadmap versions. Responded in Slack
3. Yes, the postponed item from the original doc is in **Language**

File: docs/topics/roadmap.md [hunk 1]
```diff
@@ -159,8 +162,9 @@ Visit the [new roadmap board in our issue tracker ![YouTrack](youtrack-logo.png)
         <td>
             <ul>
                 <li><a href="https://youtrack.jetbrains.com/issue/KT-42304" target="_blank">Improve IDE performance</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/TODO" target="_blank">Refactor the internal index system</a></li>
                 <li><a href="https://youtrack.jetbrains.com/issue/KT-44330" target="_blank">Improve debugging experience</a></li>
-                <li><a href="https://youtrack.jetbrains.com/issue/KT-42306" target="_blank">Improve cross-language support in the Inline Method and Change Signature refactorings</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/TODO" target="_blank">Improve the New Project wizard</a></li>
```
Comment (yalishevant) [inline docs/topics/roadmap.md]: Who is responsible for creating tickets for this and "Refactor the internal index system" items?
Should the tickets be already created?

Comment (p7nov) [reply docs/topics/roadmap.md]: @specherkin has created all the new tickets in YT, I've added links accordingly.

**Pull request merged**